### DISCUSSION TITLE
how to disable text above buttons?

### DISCUSSION THREAD

**Discussion by isaiahhoyt** [tomasklaen/uosc#438] [Q&A] [answered]
Is there anyway to disable the text that is ontop of the buttons? 
![image](https://user-images.githubusercontent.com/38567825/213895625-b299f35b-1d46-42c4-8097-480fd87a0bb4.png)

**Comment by christoph-heinrich** [comment, marked as answer]
Those are supposed to be icons, but you're missing the [fonts](https://github.com/tomasklaen/uosc/tree/main/fonts).
Also there is no way to disable them.

**Reply by isaiahhoyt** [reply]
> Those are supposed to be icons, but you're missing the [fonts](https://github.com/tomasklaen/uosc/tree/main/fonts). Also there is no way to disable them.

Yeah that fixed it, didnt know I had to put the fonts folder in the main directory, thanks.

**Reply by FichteFoll** [reply]
> Also there is no way to disable them.

Sure is, `top_bar_controls=no`.
